acceptance/tests/ensure_puppet-agent_paths.rb

```diff
@@ -66,9 +66,6 @@ def config_options(agent)
     # vardir
     {:name => :vardir,          :expected => "#{vardir}",                 :installed => :dir},
     {:name => :bucketdir,       :expected => "#{vardir}/bucket"},
-    {:name => :clientyamldir,   :expected => "#{vardir}/client_yaml",     :installed => :dir},
-    {:name => :client_datadir,  :expected => "#{vardir}/client_data",     :installed => :dir},
-    {:name => :clientbucketdir, :expected => "#{vardir}/clientbucket",    :installed => :dir},
```

Comment:
@joshcooper @jhelwig I just wanted to draw your attention to the fact that I removed these lines. They were causing the acceptance runs to fail, and it looks like we don't create any of these directories as a part of the package installation process. They are set as defaults in puppet, and previously were passing as a part of the puppet acceptance suite.

Reply:
@melissa that makes sense. Those directories are created by puppet via its settings catalog magic, and probably only get created when running `puppet agent`. I'm 👍 to only test things that we expect the MSI package to create.

Reply:
this will be for all distros, not just msis, but that sounds good to me!